app/metrics.yaml

```diff
@@ -326,4 +326,42 @@ metrics:
     notification_emails:
     - telemetry-client-dev@mozilla.com
     - fenix-core@mozilla.com
-    expires: "2020-03-01"
\ No newline at end of file
+    expires: "2020-03-01"
+
+search.default_engine:
+  code:
+    type: string
+    description: The search code for the search engine
+    send_in_pings:
+    - metrics
+    bugs:
+    - 800
+    data_reviews:
+    - TBD
+    notification_emails:
+    - fenix-core@mozilla.com
+    expires: "2020-03-01"
+  name:
+    type: string
+    description: The name of the search engine
```

Comment:
I'm concerned about the case where Fenix starts adding custom search engines.
A few thoughts here - can we add some comments, like for Search Shortcuts in #1194? Also, I'll pre-emptively file a bug for adding custom search engines, and link to both this bug and #1194 as places that we need to be careful about not tracking.